package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@reachfive/identity-core",
-  "version": "1.19.0",
+  "version": "1.19.1",
```

Comment:
I would have bumped the minor version since there's not only fixes in the PR.

Reply:
ok